ui/app/models/identity/entity-alias.js

```diff
@@ -24,6 +24,7 @@ export default IdentityModel.extend({
   }),
   metadata: attr({
     editType: 'kv',
+    isSectionHeader: true,
```

Comment:
this is already an attr option used to format `SearchSelect` with or without a large section header title, so I decided to leverage it for the `KvObjectEditor` as well

Reply:
Nice! I think this would be useful for my current KV object editor PR as well!